app/components/works/file_row_component.html.erb

```diff
@@ -3,11 +3,16 @@
     <img data-dz-thumbnail />
   </div>
   <div class="col-5">
-    <% if uploaded? %>
+    <% if saved? %>
       <div data-dropzone-target="fileName">
         <span class="fa-regular fa-file" aria-hidden="true"></span>
         <span><%= filename %>: <wbr><%= number_to_human_size(filesize) %>, <wbr><%= render LocalTimeComponent.new(datetime: date_uploaded, show_time: false) %></span>
       </div>
+    <% elsif uploaded? %>
+      <div data-dropzone-target="fileName">
+        <span class="fa-regular fa-file" aria-hidden="true"></span>
+        <span><%= filename %>: <wbr><%= number_to_human_size(filesize) %>, <wbr>Now</span>
+      </div>
     <% else %>
```

Comment:
this scenario captures when a user has uploaded a file and clicked "Save" or "Deposit" but it's not valid.  In this case, we have not yet saved to the database, but we will need the UI to render the row correctly so the user can delete that file